tools/c7n_gcp/c7n_gcp/resources/gcp_secretmanager.py

```diff
@@ -0,0 +1,18 @@
+from c7n_gcp.provider import resources
```

Comment:
minor nit, on module name, given this is already qualified by being in the gcp provider, the `gcp_` prefix seems superfluous.

Reply:
fixed

Reply:
also meant on the filename as well